src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -38,6 +38,7 @@ import NotificationType from 'consts/notificationTypes';
 import RowSaveStatus from 'consts/rowSaveStatus';
 import apiClient from 'utils/apiClient';
 import { renderFormField } from 'utils/form-utils';
+import requestsQueue from 'utils/requestsQueue';
```

Comment:
I feel like this should be handled by proper lib. Quick googling on my phone, and I land on this suggestion https://github.com/axios/axios/issues/1010#issuecomment-326172188. I think we should investigate it briefly when we are back.

Reply:
I used the TaskQueue proposed in this comment instead of the new Queue implementation